modules/cmake_configure_file/0.1.0/source.json

```diff
@@ -0,0 +1,5 @@
+{
+    "url": "https://github.com/wep21/cmake_configure_file/archive/refs/tags/v0.1.0.tar.gz",
```

Comment:
Since you are the maintainer, could you add a release archive? You can reuse this file, just manually add it as a release artifact. Then we don't need to skip the check for it.

Reply:
update url https://github.com/bazelbuild/bazel-central-registry/pull/1454/commits/c890b2df23fe0bc8fe09cf3d1c8b8b1083a788e5